lib/rules/web/admin_console/4.3/deployment.xyaml

```diff
@@ -38,6 +44,7 @@ search_nonexisted_image_and_check_message:
   action: check_page_contains
 search_and_deploy_image:
   action: search_image
+  action: choose_deploy_type_as_dc
```

Comment:
Default deploy type is Deployment while we need it to be DC, so additional action is added